.gitignore

```diff
@@ -1,4 +1,5 @@
 secret.json
+dump.rdb
```

Comment:
This shouldn't be necessary; I don't see why the database would be colocated in the code directory

Reply:
We can keep it if it's easier for you to manage the deployment. It just seems scary that the database could be overwritten with a force checkout

Reply:
I originally downloaded the secret.json and dump.rdb files into the repo dir (since I knew where secret.json needed to be), but I ended up moving the `dump.rdb` into its own dir because I needed the redis container to mount the whole data dir as opposed to just the file. However, I still like having this safeguard against whoopsie-daisy pushing the db into the cloud, for reasons such as ::gestures broadly::.